cloudigrade/api/authentication.py

```diff
@@ -95,6 +96,8 @@ class IdentityHeaderAuthentication(BaseAuthentication):
     require_account_number = True
     require_user = True
     create_user = False
+    support_account_number_header = False
+    support_org_admin_header = False
```

Comment:
Rather than take in these two specific values, I think we could better future-proof this by having one new header that we expect to have the same structure/content as the normal identity header. Then we can simplify the change in `parse_requests_header` below simply to load one header value over the other.

```suggestion
    allow_internal_fake_identity_header = False
```